src/users.php

```diff
@@ -17,15 +20,16 @@ public function __construct(&$subject, $config = array())
 		parent::__construct($subject, $config = array());
 
 		ApiResource::addIncludePath(dirname(__FILE__).'/users');
-		
-		/*load language file for plugin frontend*/ 
-		$lang = JFactory::getLanguage(); 
+
+		// Load language file for plugin frontend
+		$lang = JFactory::getLanguage();
 		$lang->load('plg_api_users', JPATH_ADMINISTRATOR,'',true);
-		
+
 		// Set the login resource to be public
 		$this->setResourceAccess('login', 'public','get');
 		$this->setResourceAccess('users', 'public', 'post');
 		$this->setResourceAccess('config', 'public', 'get');
 		$this->setResourceAccess('user', 'public', 'post');
+		$this->setResourceAccess('jfbconnect', 'public', 'post');
```

Comment:
@manojLondhe Manoj Can we have a generic resource instead of having the hard-coded JFbconnect plugin name?  example add a resource name sso and add a config in a component/plugin to choose the provider.

Reply:
@coolbung what you think?

Reply:
JFB Connect supports Google, Facebook and other OAuth providers. So you will mostly not directly speak to an OAuth provider, it will be via JFB connect. So seems fine to me to name it jfbconnect.